cgeglio/rancid-tomatillos
# Logout reset

### What does this PR do?
- Resets ratings in store when user logs out
- Adds test for remove ratings reducer
- If a user clicks "Rate" on a movie, they will be linked to the login page if they are not logged in or linked to the movie details if they are logged in

### Where should the reviewer start?
- MoviesContainer.js

### How should this be manually tested?
- Login and logout, sonic the hedgehog should no longer have a user rating
- Try to rate a movie as a logged in user and as a user that is not logged in

### Potential changes?
- If we have time, might make a popup that says 'you must be logged in to rate a movie' thennnn redirect to login

### Relevant issues?
N/A
